application/app/Http/Requests/Admin/AdminUserEditRequest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Http\Requests\Admin;
+
+use App\Models\AdminUser;
+use Illuminate\Foundation\Http\FormRequest;
+
+class AdminUserEditRequest extends FormRequest
+{
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        $rules = [
+            "name" => "required|string|max:255",
+            "email" => [
+                "required",
+                "string",
+                "email",
+                "max:255",
+                function ($attribute, $value, $fail) {
+                    $find_admin_user = AdminUser::select(["id"])
+                        ->where("email", "=", $value)->where("id", "!=", $this->admin_user->id)->first();
+                    if ($find_admin_user) {
+                        return $fail("既に登録されているメールアドレスです。");
+                    }
+                },
```

Comment:
Rule::unique(AdminUser::class)->ignore($this->admin_user)
```suggestion
               Rule::unique(AdminUser::class)->ignore($this->admin_user),
```